bindings/rust/bench/Cargo.toml

```diff
@@ -4,26 +4,45 @@ version = "0.1.0"
 edition = "2021"
 
 [features]
-historical-perf = []
+default = ["rustls", "openssl"]
+rustls = ["dep:rustls", "rustls-pemfile"]
+openssl = ["dep:openssl"]
+memory = ["plotters", "crabgrind"]
+historical-perf = ["plotters", "serde_json", "semver"]
 
 [dependencies]
 s2n-tls = { path = "../s2n-tls" }
-rustls = "0.21"
-rustls-pemfile = "1.0"
-openssl = { version = "0.10", features = ["vendored"] }
 errno = "0.3"
 libc = "0.2"
-crabgrind = "0.1"
-rand = "0.8"
-rand_distr = "0.4"
-plotters = "0.3"
-serde_json = "1.0"
-semver = "1.0"
 strum = { version = "0.25", features = ["derive"] }
+rustls = { version = "0.21", optional = true }
+rustls-pemfile = { version = "1.0", optional = true }
+openssl = { version = "0.10", features = ["vendored"], optional = true }
+crabgrind = { version = "0.1", optional = true }
+serde_json = { version = "1.0", optional = true }
+semver = { version = "1.0", optional = true }
+
+[dependencies.plotters]
```

Comment:
Rather than breaking this out into a separate table, could we keep using the inline syntax?

Reply:
We could, but the line would be way too long. Criterion does the same with breaking it out: https://github.com/bheisler/criterion.rs/blob/master/Cargo.toml#L47